subprojects/snapshots/src/main/java/org/gradle/internal/fingerprint/impl/DefaultCurrentFileCollectionFingerprint.java

```diff
@@ -36,25 +39,45 @@
     private final String identifier;
     private final FileSystemSnapshot roots;
     private final ImmutableMultimap<String, HashCode> rootHashes;
+    private final HashCode strategyConfigurationHash;
     private HashCode hash;
 
     public static CurrentFileCollectionFingerprint from(FileSystemSnapshot roots, FingerprintingStrategy strategy) {
+        return from(roots, strategy, null);
+    }
```

Comment:
This can also be removed. The test that still uses this could pass in `FileCollectionFingerprint.EMPTY` instead of `null`.

Reply:
I removed this method.

Reply:
This will need to be updated in https://github.com/gradle/dotcom/blob/786183bb944c7544ead48405957d3a8f2640a0ca/build-agent-maven-cache/src/main/java/com/gradle/maven/cache/extension/fingerprint/MojoImplementationFingerprinter.java and https://github.com/gradle/dotcom/blob/416a689e9f95fdfbce3a7a0c032bb957e7a8223e/build-agent-maven-cache/src/main/java/com/gradle/maven/cache/extension/fingerprint/MojoFingerprinter.java